src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,96 @@
+const expensesService = require('../services/expenses.service');
+const usersService = require('../services/users.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  if (!userId && !categories && !from && !to) {
+    const allExpenses = expensesService.getAllExpenses();
+
+    return res.status(200).send(allExpenses);
+  }
+
+  const expenses = expensesService.getAllExpenses(userId, categories, from, to);
```

Comment:
Consider wrapping this code into the try-catch block and sending the 500 error if something goes wrong